### ISSUE TITLE
Map does not display on live site after latest update

### ISSUE THREAD

**Issue by MiroJ** [pantheon-systems/google-map-gutenberg-block#28]
Just updated the plugin to 1.5.2 and the map does not show on the live site. I ca nee it in Gutenberg. Tried to put it again and still no code in the HTML at all. It was OK with the previous version yesterday.
I have attached screenshots. Sorry, they are not in English but you would see the map is supposed to be above the heading GPS...
![gutenberg](https://user-images.githubusercontent.com/1686322/54784302-e1475880-4bf9-11e9-8f1e-86371a251f90.png)
![html](https://user-images.githubusercontent.com/1686322/54784303-e1475880-4bf9-11e9-8060-02d2655f1012.png)
![live](https://user-images.githubusercontent.com/1686322/54784304-e1dfef00-4bf9-11e9-8475-31570fc59c8a.png)

**Comment by ataylorme** [comment]
@MiroJ thanks for the report. I'll take a look and see if I can recreate the issue.

In the meantime, you can go back to version `1.5.1`.

**Comment by ataylorme** [comment]
@MiroJ 

I just installed version `1.5.2` of the plugin on a fresh environment and saw no issues ([exmaple](https://map-block-wp-quicksilver-demo.pantheonsite.io/location/)).

Without being able to recreate the issue on my own site it is hard for me to tell what is going wrong on your site.

Maybe your issue is related to #26? Previously, the plugin was using `curl` to check if the map URL generated is valid before printing it on the front-end.

In version `1.5.2` this was updated to use the [WordPress HTTP API](https://developer.wordpress.org/plugins/http-api/).

The other change is that the check on whether the map URL is valid (if it is not the map will not print on the front-end) is cached in a [transient](https://developer.wordpress.org/reference/functions/get_transient/) so that it does not need to be checked on each page load.

Perhaps your setup does not work well with either transients or the WordPress HTTP API?
